src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -94,7 +94,7 @@ function getImagePickerOptions(type) {
 }
 
 function ReceiptSelector(props) {
-    const devices = useCameraDevices();
+    const devices = useCameraDevices('wide-angle-camera');
```

Comment:
Let's add the comment in the OP here so other engineers known why we're using the wide camera:
```suggestion
    // By default, the device with the most features gets selected, which might be a multi-camera (e.g. triple camera) - since this is not needed (we don't use zoom), we should use a simple wide angle camera instead which makes the Camera start up faster.
    const devices = useCameraDevices('wide-angle-camera');
```